qa/workunits/kernel_untar_build.sh

```diff
@@ -2,11 +2,13 @@
 
 set -ex
 
-wget -O linux.tar.gz http://download.ceph.com/qa/linux-5.4.tar.gz
+#wget -O linux.tar.gz http://download.ceph.com/qa/linux-5.4.tar.gz
+wget -O linux.tar.xz https://cdn.kernel.org/pub/linux/kernel/v6.x/linux-6.5.10.tar.xz
```

Comment:
Let's get this tarball uploaded in download.ceph.com. Also, I think we can symlink a deterministic name to this tarball (e.g., linux-kernel.tgz or whatever) so that release qa runs will automatically use the new tarball.

Reply:
@mchangir If we symlink linux-kernel.tgz to the version here, then going forward we won't require changes to release branches.

Reply:
@dmick Please upload https://cdn.kernel.org/pub/linux/kernel/v6.x/linux-6.5.10.tar.xz to download.ceph.com and place it under http://download.ceph.com/qa/ 

Also create a symlink named `linux.tar.xz` to point to it and let us know when its done.

Reply:
nvm - @mchangir let's stick to fetching the updated tarball and backport this fix (as discussed over slack).

Reply:
@dmick please ignore and drop the earlier request, thanks much

Reply:
We still need to cache the tarball in d.c.c/qa/ (just that no symlink is to be done).

Reply:
@dmick reissuing the request

Please upload https://cdn.kernel.org/pub/linux/kernel/v6.x/linux-6.5.11.tar.xz to download.ceph.com and place it under http://download.ceph.com/qa/